packages/esm-patient-appointments-app/src/appointments/appointments-form.component.tsx

```diff
@@ -107,126 +114,138 @@ const AppointmentsForm: React.FC<AppointmentsFormProps> = ({ patientUuid, isTabl
-                style={{ marginLeft: '0.125rem', flex: 'none' }}
-                labelText={t('time', 'Time')}
-                id="time-picker">
-                <TimePickerSelect
-                  onChange={(event) => onChange('timeFormat', event.target.value)}
-                  value={formData.timeFormat}
-                  labelText={t('time', 'Time')}
-                  id="time-picker-select-1">
-                  <SelectItem value="AM" text="AM" />
-                  <SelectItem value="PM" text="PM" />
-                </TimePickerSelect>
-              </TimePicker>
-            </div>
-          </section>
-          <section className={styles.formGroup}>
-            <span>{t('service', 'Service')}</span>
-            <ComboBox
-              light={isTablet}
-              onChange={({ selectedItem }) => onChange('serviceUuid', selectedItem?.uuid)}
-              id="serviceComboBox"
-              items={services}
-              itemToString={(item) => (item ? item.name : '')}
-              placeholder={t('Search for a service')}
-              titleText={t('selectServices', 'Select services')}
-            />
-          </section>
-
-          <section className={styles.formGroup}>
-            <span>{t('serviceType', 'Service Type')}</span>
-            <ComboBox
-              disabled={isEmpty(serviceTypes)}
-              light={isTablet}
-              onChange={({ selectedItem }) => onChange('serviceType', selectedItem)}
-              id="serviceTypeCombobox"
-              items={serviceTypes}
-              itemToString={(item) => (item ? item.name : '')}
-              placeholder={t('Search for a service type')}
-              titleText={t('selectServices', 'Select services type')}
-            />
-          </section>
-
-          <section className={styles.formGroup}>
-            <span>{t('appointmentKind', 'Appointment Kind')}</span>
-            <ComboBox
-              light={isTablet}
-              onChange={({ selectedItem }) => onChange('appointmentKind', selectedItem.name)}
-              id="appointmentKindCombo"
-              items={AppointmentKinds}
-              itemToString={(item) => (item ? item.name : '')}
-              placeholder={t('Select appointment kind')}
-              titleText={t('selectAppointmentKind', 'Select appointment kind')}
-            />
-          </section>
-          <section className={styles.formGroup}>
-            <span>{t('note', 'Note')}</span>
-            <TextArea
-              id="appointmentComment"
-              onChange={(event) => onChange('comments', event.target.value)}
-              light={isTablet}
-              labelText={t('writeAnAdditionalNote', 'Write an additional note')}
-              placeholder={t('writeAnAdditionalNote', 'Write an additional note')}
+    <div className={styles.formWrapper}>
+      <section className={styles.formGroup}>
+        <span>{t('location', 'Location')}</span>
+        <Select
+          id="location"
```

Comment:
Let's pick a stronger ID. These need to be globally unique, right?

```suggestion
          id="appointmentLocation"
```